src/definitions/modules/embed.js

```diff
@@ -288,6 +292,7 @@
                             .removeData(metadata.id)
                             .removeData(metadata.icon)
                             .removeData(metadata.placeholder)
+                            .removeData(metadata.alt)
```

Comment:
This does not have any effect, because there is no call to `.data(metadata.alt, somevalue)` in the whole source and removeData does not remove an existing data-attribute (see https://api.jquery.com/removedata/)
Yes, the same issue exists for metadata.placeholder and metadata.icon (bug in original SUI) as those are also nowhere set via `.data()`